services/web/server/src/simcore_service_webserver/exporter/file_downloader.py

```diff
@@ -0,0 +1,47 @@
+import sys
+import logging
+
+from aiohttp import web
+from parfive.downloader import Downloader
+from pathlib import Path
+
+from aiofiles import os as aiofiles_os
+
+from .utils import makedirs
+
+log = logging.getLogger(__name__)
+
+if sys.version_info.major == 3 and sys.version_info.minor >= 7:
+    raise RuntimeError(
+        "Upgrade parfive version to a newer one. Also remember to implement "
+        "a better check for the downloaded files. The new version has "
+        "supprot for failed and not failed futures"
+    )
```

Comment:
You are correct but I should have checked the **parfive** version, as I need to patch it to support aiofiles, to avoid blocking the loop. I will submit a PR (already talked with the maintainer). Will change this to a less confusing error and check.